packages/edit-site/src/components/post-edit/index.js

```diff
@@ -60,8 +60,24 @@ function PostEditForm( { postType, postId } ) {
 	);
 	const form = {
 		type: 'panel',
-		fields: [ 'title', 'status', 'date', 'author', 'comment_status' ],
+		fields: [
+			'title',
+			'status',
+			'date',
+			'author',
+			'slug',
+			'comment_status',
+		],
 	};
+
+	const fieldsWithBulkEditSupport = [
```

Comment:
This has a few facets and it's broken in `trunk`. I've created a new issue to discuss how to move forward at https://github.com/WordPress/gutenberg/issues/65685 Would you available to tackle that down?